README.md

```diff
@@ -1,5 +1,145 @@
 
-# Specification 'origin-policy'
+# Explainer: Origin Manifest
 
-This is the repository for origin-policy. You're welcome to contribute! Let's make the Web rock our socks
-off!
+**tl;dr** Origin Manifest (formerly Origin Policy) is a web platform mechanism
+that aims to shift configuring the origin from web applications to the origin
+itself. For this the origin provides a manifest file in a predefined well-known
+location which browsers can load and apply.
+
+**Note:** Though the current draft still uses the name Origin Policy, we use the
+new term Origin Manifest here to avoid confusion with the Same Origin Policy.
+
+
+## The Problems and Goals
+
+### Configurations with origin-wide effects
+**Problem:** Some configurations, e.g. like HTTP headers, affect an entire
+  origin even though they might have been set through only a sub-resource load.
+  Misconfiguration of font resource header can have dramatic effects for the
+  origin, e.g. a wrong HPKP value.
+
+**Goals:** Configurations for web applications with effects on entire origin in
+  a more structured way to minimize the chances for origin misconfiguration
+  through individual web applications sharing the same origin.
+
+### HTTP header redundancy
+**Problem 1:** Same HTTP headers have to be sent again and again without the
+  values actually to change. Sometimes these headers add significant number of
+  bytes, e.g. CSP.
+
+**Problem 2:** Some HTTP headers are sent with every response even though their
+  effect has a lifetime, e.g. HSTS. Within the valid time frame these headers
+  do not need to be repeated.
+
+**Goal:** Mechanism to download configurations like HTTP header values only
+  once. These values are only overridden when needed.
+
+### Missing configurations
+**Problem:** For complex web applications it is easy to forget configuring
+  certain resources or pages, e.g. the error page.
+
+**Goal 1:** Fallback settings that are applied when HTTP headers are not set.
+
+**Goal 2:** Baseline settings that guaranteed to be applied, e.g. baseline CSP
+  which guarantees a certain least level of security for the entire origin.
+
+### CORS-preflight overhead
+**Problem:** For certain resources the CORS-preflight decisions can be
+  pre-determined. That is, the server decision to (dis-)allow access is
+  independent of the actual request itself but static for certain requests, e.g.
+  "allow all requests from a.com".
+
+**Goal:** Mechanism to inform a browser about CORS access decisions beforehand
+  to avoid CORS-preflight request overhead.
```

Comment:
This seems like an instance of a more generic problem, which is that response-bound headers don't allow us to intelligently make request-time decisions. If we had a priori knowledge of a server's state for a given response, we could do some things differently. CORS preflights are one example, client hints are another. @igrigorik probably has more off the top of his head.

Reply:
certainly. The intent here was to simply name what would be nice to have, not how to solve it concretely. I know there is a bigger picture. But we cannot solve all problems but focus on our own problem(s) in this document.